### PR TITLE
show page for the menu

### PR DESCRIPTION
**Pull request by elakorkmaz**

### CODE REVIEW

File: views/show.pug [hunk 1]
```diff
@@ -7,12 +7,12 @@ block content
         img(src="/img/1.jpg")
       .col-md-2
         .row
-          h2=meal.title
+          h2=menu.title
```
Comment (izelnakri) [inline views/show.pug]: according to pug documentation you should have a space here between el= data

File: views/show.pug [hunk 2]
```diff
@@ -7,12 +7,12 @@ block content
...
           hr
         .row
           img(src="/img/heart.svg.png" width="200px" height="200px")
         .row
-          p=meal.likes
+          p=menu.votes
```
Comment (izelnakri) [inline views/show.pug]: space here too

File: views/show.pug [hunk 3]
```diff
@@ -7,12 +7,12 @@ block content
...
     .row
       .col-md-6.offset-md-3
-        p=meal.content
+        p=menu.content
```
Comment (izelnakri) [inline views/show.pug]: space here too

File: index.js [hunk 4]
```diff
@@ -2,42 +2,42 @@ const express = require('express'),
       displayRoutes = require('express-routemap'),
       morgan = require('morgan'),
       pug = require('pug'),
-      Sequelize = require('sequelize');
+      Sequelize = require('sequelize'),
```
Comment (izelnakri) [inline index.js]: you don't need sequelize dependecy here, your model already has it.

File: migrations/20161125103930-create-meal.js [hunk 5]
```diff
@@ -1,36 +0,0 @@
-'use strict';
```
Comment (izelnakri) [inline migrations/20161125103930-create-meal.js:1]: why did you remove this migration?

Comment (elakorkmaz) [reply migrations/20161125103930-create-meal.js:1]: Because when I ran a migration all the old tables were created again
